tests/example-test.sh

```diff
@@ -171,7 +171,19 @@ function waitCRDStatusReady() {
 }
 
 info "1. create kind cluster"
-make kind
+read -r -p "Is kind cluster created? [Y/n]" input
```

Comment:
Hi @huangqg ,thanks for your contribution!  This `example-test.sh` has been used to do the e2e test with the help of github actions which means this manually input will block this automation test.

Reply:
@huangqg You can use environment variables to enable(default)/disabled kind cluster creation.